addon/components/breadcrumbs.js

```diff
@@ -21,17 +21,26 @@ export default Component.extend({
    */
 
   layout,
-  tagName: 'ul',
+  tagName: '',
+  element: null,
```

Comment:
Any reason for this?

Reply:
forgot to remove it. In the first refactor I still passed the component instance to the service, and BreadcrumbsItem required a .element prop.